src/pages/komodo-defi-framework/api/legacy/sell/index.mdx

```diff
@@ -14,20 +14,20 @@ The `sell` method issues a sell request and attempts to match an order from the
+| base              | string                                | the name of the coin the user desires to sell                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                       |
+| rel               | string                                | the name of the coin the user desires to receive                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                    |
+| price             | numeric string or rational            | the amount in `rel` the user wants per one unit of the `base` coin                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                  |
+| volume            | numeric string or rational            | the maximum amount of `base` coin available for the order, ignored if max is `true`; the following values must be greater than or equal to the `min_trading_vol` of the corresponding coin: <ul><li>the argument `volume`</li><li>the product of the arguments `volume` and `price`</li></ul>                                                                                                                                                                                                                                                                       |
```

Comment:
> available for the order

Available for what? Sending or receiving?
Please use marker verbs, in sell `volume` represents amount of `base` coin user is willing to send.
"Send"/"receive" are easier to understand for non-native speakers and easier to spot at a glance without reading the full sentence.

Reply:
updated to `the amount of `rel` coins the user wants to receive for each unit of the `base` coin spent.` (same as setprice)

Reply:
updated to `the maximum amount of `base` coin available for sale in the order`